src/pyrokinetics/gk_code/gene.py

```diff
@@ -384,18 +384,24 @@ def get_local_geometry(self) -> LocalGeometry:
                 local_geometry_data["q"] ** 2 * local_geometry_data["Rmaj"]
             )
         else:
-            amhd_beta_prime = (
-                -local_species.inverse_lp.m
-                * local_species.pressure.m
-                / local_geometry_data["B0"] ** 2
-            )
+            if local_geometry_data["B0"] is not None:
+                amhd_beta_prime = (
+                    -local_species.inverse_lp.m
+                    * local_species.pressure.m
+                    / local_geometry_data["B0"] ** 2
```

Comment:
Thinking about this, we could just multiple it by `beta` rather than dividing by `1/B0^2` as `beta` is always set even if not there